### ISSUE TITLE
Julia and Python multiline strings in code block doe not highlight correctly

### ISSUE THREAD

**Issue by oxinabox** [marktext/marktext#1719]
### Description

Inside fenced codeblocks for the python or julia language,
syntax highlighting does not work on triple quoted strings.

- [ ] Can you reproduce the issue?

### Steps to reproduce

1. Open program
2. Create a fenced code block, speciifying language as julia or python
3. enter a triple quoted multiline string

**Expected behavior:**

Nice syntax highlighting of a string,
e.g as on github:

```python
"""
Python has 
multiline strings
that also do not highlight right
"""
```

**Actual behavior:**

(Input text at bottom of post)
<img width="705" alt="image" src="https://user-images.githubusercontent.com/5127634/69015076-774da180-0988-11ea-93ce-d39fc852eeba.png">

### Versions

- Mark Text: 0.15.0
- Operating system: Mac 10.15.1

---

```julia
    """
    Julia has 
    Triple quoted strings 
    that do not highlight right
    """
    ```

    SImilar in python

    ```python
    """
    Python has 
    multiline strings
    that also do not highlight right
    """
    ```

**Comment by Jocs** [comment]
@oxinabox Can you help test this issue on version 0.16.0-rc.1, because I remove this is already be fixed.

**Comment by oxinabox** [comment]
> @oxinabox Can you help test this issue on version 0.16.0-rc.1, because I remove this is already be fixed.

I cannot. If its fixed then by all means close this.

**Comment by adrienbeau** [comment]
I just tested it. This is fixed in 0.16.0-rc1.
